docs/pages/identity-governance/access-lists/terraform.mdx

```diff
@@ -0,0 +1,314 @@
+  metadata = {
+    name = "db_access_staging"
+  }
+  spec = {
+    allow = {
+      db_users = ["viewer"]
+      db_names = ["*"]
+      db_labels = {
+        env = "staging"
+      }
+    }
+  }
+}
+
+resource "teleport_access_list" "db_access_staging" {
+  header = {
+    version = "v1"
+    metadata = {
+      name = "db_access_staging"
+    }
+  }
+  spec = {
+    type = "static" # static, so members can be assigned with Terraform
+    title       = "Staging DBs access"
+    description = "Read-only access to staging databases"
+    owners = [
+      { name = "admin" },
+    ]
+    grants = {
+      roles = [teleport_role.db_access_staging.id]
+    }
+  }
+}
+```
+
+
+## Step 3/5. Give access to the team
+
+Knowing that nested Access Lists inherit grants of the parent Access list, giving "Developers" team
+"Staging DBs access" is now as simple as creating a nested Access List membership:
+
+```hcl
+resource "teleport_access_list_member" "db_access_staging_developers" {
+  header = {
+    version = "v1"
+    metadata = {
+      name = teleport_access_list.developers.id
+    }
+  }
+  spec = {
+    access_list     = teleport_access_list.db_access_staging.id
+    membership_kind = 2 # 1 for "MEMBERSHIP_KIND_USER", 2 for "MEMBERSHIP_KIND_LIST"
+  }
+}
+```
+
+Note, `membership_kind` is not `2` (MEMBERSHIP_KIND_LIST), because this is nested Access List
+membership.
+
+Once "Developers" Access List is a member of "Staging DBs access" Access Lists, "Developers" list
+inherits grants of the "Staging DBs access" and in effect developers are granted access to the
+staging databases.
+
+
+## Step 4/5. Give access to a team managed in the web UI
+
+This step will show how Access Lists of the default type can be employed in the structure. As a
+quick reminder, the default Access Lists are supposed to be managed with the web UI and they
+require periodic reviews performed by the list owners. The default Access List members cannot be
+manged with Terraform, but the lists themselves can be created/modified and imported to Terraform.
+
+Let's create a default Access List representing another team "DB Administrators" reusing the "noop"
+role created in the previous steps:
+
+```hcl
+resource "teleport_access_list" "dbas" {
+  depends_on = [ teleport_role.noop ] # roles can't be deleted if they are in use
+  header = {
+    version = "v1"
+    metadata = {
+      name = "dbas"
+    }
+  }
+  spec = {
+    # type is skipped, this is a regular Access Lists
+    title       = "DBAs"
+    description = "DB Administrators team."
+    owners = [
+      { name = "admin" },
+    ]
+    grants = {
+      roles = ["noop"]
+    }
+    # audit settings can be skipped, the values below are the defaults
+    audit = {
+      recurrence = {
+        frequency    = 6 # every 6 months
+        day_of_month = 1 # first day of the month
+      }
+    }
+  }
+}
+```
+
+The new "DBAs" team members can be managed using web UI. Teleport user "admin", as the Access List
```

Comment:
We should use `code style` instead of double quotes for user names to make it clear that these have been specified in a configuration somewhere in this guide.